docker-compose.yml

```
@@ -1,5 +1,4 @@
 name: localcofhenix
```

well, I think it's getting official so...
```suggestion
name: localcofhe
```